### PR TITLE
chore: add rule to .editorconfig rule set

### PR DESCRIPTION
**Pull request by SSelinn**
## Description

This PR makes sure all fields are formatted like this: _doSomething

## Testability

If you open up a class and declare some variable: string Name, it should suggest to do _name.

## Checklist
- [x] Set the proper pull request name
- [x] Merged main into your branch

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -254,13 +254,13 @@ tab_width = 4
...
-dotnet_naming_symbols.private_fields.applicable_kinds           = field
-dotnet_naming_symbols.private_fields.applicable_accessibilities = internal, protected_internal, private 
+# Fields are camelCase and start with _
+dotnet_naming_rule.fields_should_be_camel_case.severity = suggestion
+dotnet_naming_rule.fields_should_be_camel_case.symbols = underscore_fields
+dotnet_naming_rule.fields_should_be_camel_case.style = underscore_field_style
 
-dotnet_naming_style.prefix_underscore.capitalization  = camel_case
-dotnet_naming_style.prefix_underscore.required_prefix = _
+dotnet_naming_symbols.underscore_fields.applicable_kinds = field
```
Comment (JensSteenmetz) [inline .editorconfig]:
```suggestion
dotnet_naming_symbols.underscore_fields.applicable_kinds = field, property
```

File: .editorconfig [hunk 2]
```diff
@@ -254,13 +254,13 @@ tab_width = 4
...
+dotnet_naming_symbols.underscore_fields.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
```
Comment (JensSteenmetz) [inline .editorconfig]:
```suggestion
dotnet_naming_symbols.underscore_fields.applicable_accessibilities = internal, private, protected, protected_internal, private_protected
```

File: .editorconfig [hunk 3]
```diff
@@ -254,13 +254,13 @@ tab_width = 4
 indent_size = 4
 end_of_line = crlf
 
-# Use underscores for private fields
-dotnet_naming_symbols.private_fields.applicable_kinds           = field
-dotnet_naming_symbols.private_fields.applicable_accessibilities = internal, protected_internal, private 
+# Fields are camelCase and start with _
+dotnet_naming_rule.fields_should_be_camel_case.severity = suggestion
```
Comment (JensSteenmetz) [inline .editorconfig]:
```suggestion
dotnet_naming_rule.fields_should_be_camel_case.severity = warning
```

**Review by JensSteenmetz — CHANGES_REQUESTED** [review]
I think it should be like this?

**Review by JensSteenmetz — CHANGES_REQUESTED** [review]
Actually, we might need to discuss this tomorrow, as there are some more issues that require some decisions regarding style.

**Review by QuakeEye — DISMISSED** [review]

**Review by Tboefijn — DISMISSED** [review]

**Comment by joachimdekker** [comment]
Due to already implementing it in another PR, this PR does not have anything to add. Sorry. Thus, I am closing the PR.

**Pull request closed without merging**